assets/javascripts/birdeck.js

```diff
@@ -21,9 +21,7 @@ function renderPost(post) {
 }
 
 function fetchPostsButton() {
-  $("#button-fetch-posts").on("click", function(){
-    fetchPosts()
-    })
+  $("#button-fetch-posts").on("click", fetchPosts)
```

Comment:
@novohispano: You're creating an entirely pointless object and adding an entire entry to the call stack. For what reason?

We have to spend an unreasonable amount of time cleaning up the half-taught concepts in Module 3 that then get reinforced when they use these concepts for their self-directed projects.